liblwgeom/lwgeom_geos.c

```diff
@@ -495,493 +488,316 @@ lwgeom_geos_version()
 	return ver;
 }
 
-LWGEOM*
-lwgeom_normalize(const LWGEOM* geom1)
+inline static const uint32_t
```

Comment:
`const` here is superfluous

Reply:
Also SRIDs can be negative, so `int32_t` is preferable (geom->srid already is int32_t).

Reply:
thanks, fixed.